### ISSUE TITLE
Register Function

### ISSUE THREAD

**Issue by corinloch** [sn0w12/Akari#3]
Is there any, to have a register on the akari site itself, without going to Manganeto? Thanks!

**Comment by sn0w12** [comment]
Not currently, but i'll look into it at some point for sure. I don't think the process is too different from logging in. I have some other features I'm looking into right now but i'll probably come around to it pretty soon.

**Comment by corinloch** [comment]
Thanks bro, Also bro where can I edit the main homepage layout? wanna try something, Thanks!

**Comment by sn0w12** [comment]
Most of it is found at: `src/components/MangaReaderHome.tsx`.

**Comment by corinloch** [comment]
I encountered this error while following the installation, please help and also is it possible to log-in via discord? Thanks!

Error:  An error occurred Failed to load manga data: Error: Failed to fetch manga data: 404 Not Found

**Comment by sn0w12** [comment]
Generally a 404 error means the manga doesn't exist, I can't really help you in any way just from that. You will need to describe the issue in more detail. I do not plan on adding any other login methods.

**Comment by corinloch** [comment]
I did tried on my other vps and it works I think it's a port issue? on my main vps I tried using a different port which is 3016 but on the terminal it says this:

status : 404
url:http://localhost:3000/api/manga-list/latest?page=1

how can I make it it use 3016? I cant't use the default port 3000 as I am currently using it, Thanks!

**Comment by corinloch** [comment]
Thanks, it's now working and now I have my own private manga site, Thanks bro!

**Comment by corinloch** [comment]
One more thing is it possible to make the register on the website not for manganato as login is working? kinda annoying if need to go third party site as some of my friends has uorigin and manganato is blocked by default, if possible how? Thanks!

**Comment by sn0w12** [comment]
Manganato is used to track the users bookmarks so it's not really possible, but i'll try and add an option to register without having to go to another site.

**Comment by corinloch** [comment]
Yeah that register without having to another site is godsend, let me know if it's available. Thanks!

**Comment by sn0w12** [comment]
There is now a register page, you can find it when trying to log in.
